enclave-manager/web/src/client/enclaveManager/KurtosisClient.ts

```diff
@@ -74,9 +91,47 @@ export abstract class KurtosisClient {
   }
 
   async checkHealth() {
+    console.log(await this.restClient.GET("/engine/info"));
     return asyncResult(this.client.check({}, this.getHeaderOptions()));
   }
 
+  async *getServiceLogsWS(
+    abortController: AbortController,
+    enclave: RemoveFunctions<EnclaveFullInfo>,
+    serviceUUID: string,
+    followLogs?: boolean,
+    numLogLines?: number,
+    returnAllLogs?: boolean,
+    conjunctiveFilters?: LogLineFilter[]
+  ): AsyncGenerator<components["schemas"]["ServiceLogs"]> {
```

Comment:
It was dropped when my commit was rolled back, but having a type like `type KurtosisRestTypes = components["schemas"]` would be slightly better to use throughout the code. Importing a type called `components` everywhere we need them is misleading as:
* The reader probably expects it to contain React components
* Lowercase name for a type is not consistent with style
* Unclear what `components` the thing is referring to

Reply:
Yes, using a type alias for this will be much clear.